core/src/main/java/com/chopshop166/chopshoplib/motors/CSTalonBase.java

```diff
@@ -80,4 +80,14 @@ public TalonEncoder getEncoder() {
     public void setSetpoint(final double setPoint) {
         this.wrapped.set(this.savedControlType, setPoint);
     }
+
+    @Override
+    public double[] getCurrentAmps() {
+        return new double[] {this.wrapped.getStatorCurrent()};
```

Comment:
This was definitely the wrong choice. Need to update this to use the input current.